src/Authentik/README.md

```diff
@@ -0,0 +1,43 @@
+# Authentik
+
+```bash
+composer require socialiteproviders/authentik
+```
+
+## Installation & Basic Usage
+
+Please see the [Base Installation Guide](https://socialiteproviders.com/usage/), then follow the provider specific instructions below.
```

Comment:
Can we add a link to the authentik docs?

Reply:
leaving here Oauth2 docs for authentik, just in case https://goauthentik.io/docs/providers/oauth2/ :)